CODEOWNERS

```diff
@@ -6,6 +6,5 @@
 #   https://blog.github.com/2017-07-06-introducing-code-owners/
 #   https://help.github.com/articles/about-codeowners/  
 
-# Default owner for repo
-*	@pragnagopa
-*   @amamounelsayed
\ No newline at end of file
+# Default owners for repo
+*	@Azure/azure-functions-java-worker-admins
```

Comment:
is the leading space causing problem?
It seems only Mamoun's alias was added and not Pragna's previously because there is a leading space in front of her alias.

- Check same for other repo too.

Reply:
Looks like it is not working, 
<img width="535" alt="image" src="https://github.com/user-attachments/assets/c2eb4f0a-a6e1-4d5c-b9d4-f0e63923f8a5">
Hovering over the error message suggests that the team is not being found or not visible publicly

Reply:
Updated to correct role